Svg icons

TobiTenno opened the pull request:
### Warframe Hub Pull Request
---
---

**Summary (short):** Using svgs for resources where possible. See deployment for how it looks



(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

Tithen-Firion commented on assets/css/main.css:
```diff
@@ -167,6 +167,11 @@ img {
   margin-top: -3px;
+}
+
+.nightmare:not([src*="/svg/"], 
```
I think we should keep it simple, just invert image colors so png matches svg.

TobiTenno replied:
like make all the pngs black?

TobiTenno replied:
i'll do a pass tonight when i get home

TobiTenno committed:
add svgs, need to swap in code
Add files via upload
update wfcd logo to use an svg
platform svgs
add invert class to properly invert on nightmode
add invert to properly invert on darker modes
svg images with png fallback
Add files via upload
adding paths for corrupted, need to fix image locations
Add files via upload
Delete grineer.png
Delete infested.png
Delete corpus.png
Add files via upload
Fix the Warframe Community Devs org banner link (#85)

* Fix the Warframe Community Devs org banner link

* change banner to wfcd

Merge branch 'dev' into svg-icons
syntactic fixes
svg inversion fixes
max size for faction image
fix line, force height and width in html attr's
use getImage so only one source is ever referenced

linting

linting

fissures path fix

factions image path fix

css path fix

linting

standing common css

add invert by default for svgs

move common stuff to common.css

move common stuff to common.css

move common stuff to common.css

fix some inversion weirdness

force progress bar faction images white

retro fixes

retro fixes

put common css before custom mode and after main mode

global theme switch handling

attempt to fix odd theme behavior

consolidate theme files

consolidate theme files

Move eidolon theme to unified theme handler


fix merge flubs


picker coloring weirdness


fix dropdown picker's image misalignment

TobiTenno committed:
bounty expando, still need to figure out "unidentified" above panel

TobiTenno committed:
silly me, lets start as undefined.

TobiTenno committed:
Fix misplaced open list item around void trader

wfcd-bot-boi commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on:
- [GitHub release](https://github.com/WFCD/warframe-hub/releases/tag/v1.0.0)
- `v1.0.0`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
